src/openrct2/ride/RideData.cpp

```diff
@@ -49,6 +49,7 @@
 #include "coaster/meta/LIMLaunchedRollerCoaster.h"
 #include "coaster/meta/LayDownRollerCoaster.h"
 #include "coaster/meta/LoopingRollerCoaster.h"
+#include "coaster/meta/Maurer-Gerstlauer Spinner RollerCoaster.h"
```

Comment:
We dont use spaces or hyphens in file names.

Reply:
ok so i can change it to MaurerGerstlauuerSpinnerRollercoaster.h

Reply:
Yeah don't use a manufacturer name in general. You'll see all the other coaster names are generic.

Reply:
yes true maybe because of the copyright rights also i will change that

Reply:
Yeah it should be **SteelSpinningRollerCoaster**

Reply:
![grafik](https://github.com/OpenRCT2/OpenRCT2/assets/77458610/92539dc9-3404-4584-be5a-f2351b0c7f8c)
Now I changed all the names now its generic :)

finally we have the name now